From 750d6ec0d4e984cf1d2f3c53f498d86365e91ed5 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 11 May 2018 16:27:30 +0800
Subject: [PATCH 3778/5242] MLK-18277-02 driver: soc: imx: add power domain
 compatible for imx8mm

commit  c55b71f4964800c42879ba921d51a9b37d9d5e64 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add power domain compatible string for i.MX8MM.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/imx/gpc-psci.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/imx/gpc-psci.c b/drivers/soc/imx/gpc-psci.c
index 701a708..593fb14 100644
--- a/drivers/soc/imx/gpc-psci.c
+++ b/drivers/soc/imx/gpc-psci.c
@@ -330,6 +330,7 @@ static int imx_gpc_pm_domain_probe(struct platform_device *pdev)
 
 static const struct of_device_id imx_gpc_pm_domain_ids[] = {
 	{.compatible = "fsl,imx8mq-pm-domain"},
+	{.compatible = "fsl,imx8mm-pm-domain"},
 	{},
 };
 
-- 
1.7.9.5

